tools/convert.rb

```diff
@@ -854,7 +854,7 @@ def parseUCIngest(itemID, inMeta, fileType)
   issue = section = nil
   volNum = inMeta.text_at("./context/volume")
   issueNum = inMeta.text_at("./context/issue")
-  if issueNum or volNum
+  if inMeta[:state] != "withdrawn" and (issueNum or volNum)
```

Comment:
It just struck me to wonder about embargoed items. Out of scope for the story we're working on though.